adcircpy/_fort15.py

```diff
@@ -293,7 +292,7 @@ def write(self, runtype, path, overwrite=False):
         if fort15.exists() and not overwrite:
             msg = f"{fort15} exists. Pass overwrite=True to overwrite."
             raise Exception(msg)
-        with open(fort15, 'w') as f:
+        with open(fort15, 'w', newline='\n') as f:
```

Comment:
fort.15 should not required this

Reply:
I added this just to be safe, I can remove if you'd like

Reply:
I don't think this will have any effect, but it's fine.